<?xml version="1.0"?>
<launch>

  <arg name="sensor_name" default="camera" />
  <arg name="use_pointcloud" default="true" />

  <!-- Launch object detection node -->
  <node pkg="recognition" type="tvm_hand_detection_node"
        name="$(arg sensor_name)_tvm_hand_detection_node" output="screen" respawn="false">

    <param name="sensor_name"               value="$(arg sensor_name)" />
    <param name="world_frame_id"            value="/world"/>
    <param name="depth_image_topic"         value="/$(arg sensor_name)/depth/image_rect_raw"/>
    <param name="rgb_image_topic"           value="/$(arg sensor_name)/color/image_rect_color"/>
    <param name="output_topic"              value="/objects_detector/detections"/>
    <param name="camera_info_topic"         value="/$(arg sensor_name)/color/camera_info"/>

    <param name="root"                      value="$(find recognition)"/>

  </node>

</launch>